metis/lib/models/data_block.rb

```diff
@@ -34,6 +34,14 @@ def set_file_data(file_path, copy=false)
       end
     end
 
+    def removed?
```

Comment:
Given the conversation around keeping `removed` as a separate column, should the changes in this file be reverted?